docs/configuration-reference/platforms/packet.md

```diff
@@ -240,3 +241,37 @@ To destroy the Lokomotive cluster, execute the following command:
 lokoctl cluster destroy --confirm
 ```
 
+## ARM support and hybrid clusters
+
+Lokomotive and Flatcar currently include an alpha-quality preview for the
+Packet arm64 server types `c1.large.arm` and `c2.large.arm`. They can be used
```

Comment:
I'm not sure if `c1.large.arm` still works, because we did not try it with Flatcar anymore for long time and last time I tried, it did not come up.

Reply:
I'll give it a try

Reply:
It doesn't seem to get past grub:

```
Booting `Flatcar default'

EFI stub: Booting Linux Kernel...
EFI stub: EFI_RNG_PROTOCOL unavailable, no randomness supplied
EFI stub: Using DTB from configuration table
EFI stub: Exiting boot services and installing virtual address map...
```

I'll remove it.